po/lt.po

```diff
@@ -3207,129 +3207,234 @@ msgstr "Rašau %s..."
 
 #. L10N: Try to limit to 80 columns
 #: main.c:153
+msgid "usage:"
+msgstr "usage:"
```

Comment:
Well, at least one good thing came out of this PR: I noticed that this bit was never translated!